src/node_buffer.h

```diff
@@ -19,9 +19,25 @@ NODE_EXTERN bool HasInstance(v8::Local<v8::Value> val);
 NODE_EXTERN bool HasInstance(v8::Local<v8::Object> val);
 NODE_EXTERN char* Data(v8::Local<v8::Value> val);
 NODE_EXTERN char* Data(v8::Local<v8::Object> val);
+
+#if ENABLE_TTD_NODE
+NODE_EXTERN void TTDAsyncModRegister(v8::Local<v8::Object> val,
+    byte* initialModPosition);
+NODE_EXTERN void TTDAsyncModNotify(byte* finalModPosition);
+NODE_EXTERN void TTDSyncDataModNotify(v8::Local<v8::Object> val,
+    UINT32 index, UINT32 count);
+#endif
+
 NODE_EXTERN size_t Length(v8::Local<v8::Value> val);
 NODE_EXTERN size_t Length(v8::Local<v8::Object> val);
 
+#if ENABLE_TTD_NODE
+// Notify us that a native buffer access (which we don't currently understand/support) happened.
+#define TTD_NATIVE_BUFFER_ACCESS_NOTIFY(X) JsTTDCheckAndAssertIfTTDRunning("Unsupported raw buffer access -- investigate this!!!\n")
```

Comment:
Lines are > 80 char, please run cpplint on all changed .cc and .h files and eslint on .js files.